### ISSUE TITLE
Remove dependency on Bristol installation

### ISSUE THREAD

**Issue by benkrikler** [alphatwirl/heppy-to-dataframe-demonstrator#4]
The demo was developed on the Bristol HEP group's computing space, but needs to become site agnostic. 

The setup.sh file has the following issues:
 * No check that a specific data file exists -- could add other paths for other common machines?
 * Source's a setup script hosted on CVMFS and therefore breaking on Laptops, etc.

In addition, step7 depends on htcondor, but to fix or replace this would need interfaces to other batch systems to be included, which is a bigger task for something alphatwirl itself.

**Comment by kreczko** [comment]
A solution would be to make it work on lxplus (EOS + CVMS + HTCondor).
If you use `root://` paths, it might even work at Bristol. I can certainly help with that.

**Comment by benkrikler** [comment]
Should also remove [these lines](https://github.com/alphatwirl/heppy-to-dataframe-demonstrator/blob/master/step4_heppy_interface/run.py#L141-L142) so that we can run on heppy output structures that don't contain the currently hard-coded components.

**Comment by benkrikler** [comment]
>  If you use root:// paths, it might even work at Bristol. I can certainly help with that.

Yeah, it could be nice to suggest a decent path for that in the README. 
For working on a laptop, we could go via the vagrant route, or perhaps I add a setup.sh file that uses pip with a list of requirements instead?  We can do both as well of course, the goal being to reduce the things people to have to manually install to get running.

**Comment by benkrikler** [comment]
Also, @kreczko, perhaps we could add a vagrantfile to this for non-RHL users to be able to play?

**Comment by kreczko** [comment]
Yes, Vagrantfile is not a bad idea. You would still need to solve the input file problem though. I will check tomorrow/Sunday.
